test/langtools/tools/javac/launcher/SourceLauncherTest.java

```diff
@@ -598,6 +598,38 @@ public void testClassInModule(Path base) throws IOException {
                 "error: compilation failed");
     }
 
+    @Test
+    public void testNoRecompileWithSuggestions(Path base) throws IOException {
+        tb.writeJavaFiles(base,
+            "class NoRecompile {\n" +
```

Comment:
It is better to use text block `"""` to improve the readability.

Reply:
Right, however it will prevent clean backports to older JDK versions